packages/_vue3-migration-test/src/x-modules/facets/components/test-facets.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div>
+    <h2>Facets</h2>
+    <Facets :renderableFacets="facetsIds.join(',')">
+      <template #default="{ facet }">
+        <h3>{{ facet.label }}</h3>
+        <ul>
+          <li v-for="filter in facet.filters" :key="filter.id">
```

Comment:
```TS
<li v-for="filter in facet.filters" :key="filter.id">
  <BaseEventButton :events="{ UserClickedAFilter: filter }">
    {{ filter.label }}
  </BaseEventButton>
</li>
```

WDYT to be able to select and deselected for the time being until filter migrations?